uk.co.compucorp.civicrm.tasksassignments/views/dashboard/key-dates.html

```diff
@@ -47,7 +47,7 @@ <h3 class="panel-title">View Dates By:</h3>
                             </div>
                         </div>
                         <button class="btn btn-default btn-primary btn-block"
-                                ng-click="labelDateRange(filterParams.dateRange.from, filterParams.dateRange.until);
+                                ng-click="labelDatZXCeRange(filterParams.dateRange.from, filterParams.dateRange.until);
```

Comment:
ops! thanks for catching! also the indentation of the file should be fixed. Will do.